CHANGELOG.md

```diff
@@ -1,5 +1,33 @@
 # Change Log
 
+## [v1.1.4](https://github.com/ably/ably-go/tree/v1.1.4) (2020-04-24)
+
+[Full Changelog](https://github.com/ably/ably-go/compare/v1.1.3...v1.1.4)
+
+**Implemented enhancements:**
+
+- Implement host fallback [\#53](https://github.com/ably/ably-go/issues/53)
```

Comment:
I just closed the issue, but this was done at #78 two years ago.

Reply:
Well spotted. Thanks! I don't want to start retroactively trying to modify the old change history but equally removing this line from this release PR may end up confusing the change log generator for future releases (actually, perhaps not using the delta option that I prefer using these days!). So, yes, I'll just remove the line as it's not helpful to those looking or a 'human readable' change log that tells them what's new! `</ monologue>`

Reply:
https://github.com/ably/ably-go/pull/158/commits/694511d0905b06c1af23f6026008268641a9eba0